layers/state_tracker/cmd_buffer_state.cpp

```diff
@@ -1078,7 +1070,20 @@ void CMD_BUFFER_STATE::UpdatePipelineState(CMD_TYPE cmd_type, const VkPipelineBi
     const auto lv_bind_point = ConvertToLvlBindPoint(bind_point);
     auto &last_bound = lastBound[lv_bind_point];
     PIPELINE_STATE *pipe = last_bound.pipeline_state;
-    if (pipe && VK_NULL_HANDLE != last_bound.pipeline_layout) {
+    if (!pipe) {
+        return;
+    }
+
+    // Update the consumed viewport/scissor count.
+    {
+        uint32_t &used = usedViewportScissorCount;
+        used = std::max(used, pipelineStaticViewportCount);
+        used = std::max(used, pipelineStaticScissorCount);
```

Comment:
std::max can take a initializer_list

```suggestion
        usedViewportScissorCount = std::max({usedViewportScissorCount, pipelineStaticViewportCount, pipelineStaticScissorCount});
```

Reply:
thanks, I just moved this, but like fixing it while here